Example/Example/AppDelegate.swift

```diff
@@ -16,8 +16,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
         _ = SentrySDK(options: [
-            "dsn": "",
-            "debug": true
+            "dsn": "https://8ee5199a90354faf995292b15c196d48@o19635.ingest.sentry.io/4394",
+            "debug": true,
+            "enableSessionTracking": true,
+            "sessionTrackingIntervalMillis": 5000 // 5 seconds session timeout for testing
```

Comment:
We should just remove the DSN before merging.

Reply:
In other SDKs (Android, .NET) we have a DSN that goes to `sentry-test` to make it easier for the maintainers. README points out users should change in order to get sample events on their dashboards. I personally like that since 99% of the runs are from someone who has access to `sentry-test` org.